app/assets/stylesheets/govuk_publishing_components/components/govspeak/_button.scss

```diff
@@ -24,4 +24,33 @@
   .govuk-button--start {
     @extend .govuk-button--start;
   }
+
+  // stylelint-disable declaration-no-important
+  @include govuk-media-query($media-type: print) {
+    .gem-c-button {
+      display: inline-block;
+      box-shadow: none;
+      border: 2pt solid $govuk-print-text-colour;
+
+      &,
+      & * {
+        background-color: unset;
+        color: $govuk-print-text-colour !important;
+      }
+
+      &::before {
+        display: none;
+      }
+
+      &::after {
+        display: none;
```

Comment:
If the after element is `display: none` why set any other attributes, like position?

Reply:
This isn't even needed as there's no `::after` element in the markup. I think this was mistakenly copied from another component. I've deleted it.